members/models/address.py

```diff
@@ -36,18 +38,20 @@ class Meta:
         "Breddegrad", blank=True, null=True, max_digits=9, decimal_places=6
     )
     help_temp = """
-    Lader dig gemme en anden Længdegrad og breddegrad end den gemt i DAWA \
+    Lader dig gemme en anden Længdegrad og Breddegrad end oplyst fra  DAWA \
```

Comment:
Mindre sproglig rettelse, du kan vælge at tage med eller ignorere 🙂 
```suggestion
    Lader dig gemme en anden længde- og breddegrad end oplyst fra DAWA \
```
(I fald vi retter, skal vi også lige rette migration fil)